### PR TITLE
Lists navigation components

### PR DESCRIPTION
**Pull request by b0llu**
list and navigation components are merge ready please give feedback
image
image
image

### CODE REVIEW

File: Components/lists/lists.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  border-radius: 0.5rem;
+}
+
+.notification-list li {
+  padding: 1rem 1.5rem;
+  margin: 1rem 1rem;
+  border: 1px solid #e6e6e9;
+  border-radius: 0.5rem;
+  background-color: #f7f5f5;
+}
```
Comment (Maianki) [inline Components/lists/lists.css:47]: You can use CSS variables for colors to be consistent.

**Review by Maianki — CHANGES_REQUESTED** [review]
I have suggested a small change. Everything else looks good.

File: Components/lists/lists.css (hunk 1 above)
Comment (b0llu) [reply Components/lists/lists.css:47]: thanks for that! actually this is the only place i am using this colors so used hex instead.

Comment (Maianki) [reply Components/lists/lists.css:47]: Okay, Everything else looks good.

**Review by Maianki — APPROVED** [review]
LGTM

File: Components/lists/lists.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  line-height: 3rem;
+  border-radius: 0.5rem;
+}
+
+.notification-list li {
+  padding: 1rem 1.5rem;
+  margin: 1rem 1rem;
+  border: 1px solid #e6e6e9;
+  border-radius: 0.5rem;
+  background-color: #f7f5f5;
```
Comment (arjun-computer-geek) [inline Components/lists/lists.css:46]: I think it should be used as a color variable. What do you think?

**Review by arjun-computer-geek — APPROVED** [review]
Now it is looking good to me

File: Components/lists/lists.css (hunk 2 above)
Comment (b0llu) [reply Components/lists/lists.css:46]: yes it should be but I am only using these colors in this spot specifically. thats the reason for not using a variable. but thanks for the review

**Pull request merged**